modules/modules.sqf

```diff
@@ -14,13 +14,15 @@
 #include "start_on_safe\root.sqf"
 #include "start_with_earplugs\root.sqf"
 
-//Additional modules that can be enabled by removing the // 
+//Additional modules that can be enabled by removing the //
 //!!Some of them require changing settings in their settings.sqf file!!
 
 //#include "a3_endscreen\root.sqf"
+//#include "aceMedical\root.sqf"
+//#include "ai_attack\root.sqf"
 //#include "ai_behaviour\root.sqf"
 //#include "ai_drivers\root.sqf"
-//#include "ai_link\root.sqf"
+#include "ai_link\root.sqf"
```

Comment:
I recommend disabling it by default since it's complexity is O(n^2) (where n = number of groups) every 15 seconds. number of function calls can go up to 60 000.